vehicle-model-lifecycle/src/generate_model.py

```diff
@@ -104,7 +94,7 @@ def install_model_if_required(language: str, model_path: str) -> None:
         model_path (str): The path where the generated model is stored.
     """
     if language == "python":
-        subprocess.check_call([sys.executable, "-m", "pip", "install", model_path])
+        subprocess.check_call(["sys.executable", "-m", "pip", "install", model_path])
```

Comment:
Why is `sys.executable` quoted now? That way its just a string literal and not a variable which is being resolved. I doubt this change is runnable. Have you tested it?